migration_tools.drush.inc

```diff
@@ -248,6 +263,51 @@ function drush_migration_tools_mt_migrate_file_folders($organization) {
 }
 
 /**
+ * Drush command callback for migration-tools-html-file-list.
+ *
+ */
+function drush_migration_tools_html_file_list($directory) {
+  // Set up options
+  $format = drush_get_option('format');
+  $outfile = drush_get_option('outfile');
```

Comment:
maybe add this in case an outfile is requested but not csv.  Then the format can be used for the filename. (down farther)
```
$format = (empty($format)) ? 'txt' :  $format;
```